traffic_ops/traffic_ops_golang/staticdnsentry/staticdnsentry.go

```diff
@@ -106,7 +107,7 @@ func (staticDNSEntry TOStaticDNSEntry) Validate() error {
 	}
 
 	errs := validation.Errors{
-		"host":              validation.Validate(staticDNSEntry.Host, validation.Required),
+		"host":              validation.Validate(staticDNSEntry.Host, validation.Required, validation.Match(regexp.MustCompile(`^([a-zA-Z0-9_@*]{1}[a-zA-Z0-9_-]{0,62}){1}(\.[a-zA-Z0-9_]{1}[a-zA-Z0-9_-]{0,62})*[\._]?$`))),
```

Comment:
you can just use [`github.com/go-ozzo/ozzo-validation/is.DNSName`](https://godoc.org/github.com/go-ozzo/ozzo-validation/is#pkg-variables) instead of making your own regular expression

Reply:
no, he can't because that regex doesn't include `@` or `*`

Reply:
`*` I get but what does `@` do?

Reply:
https://help.dyn.com/how-to-format-a-zone-file/